minimal_Sage_standard.json

```diff
@@ -130,8 +132,43 @@
         "dorsolateral prefrontal cortex",
         "anterior cingulate cortex",
         "temporal cortex",
-        "serum",
-        "Not Applicable"
+        "orbitofrontal cortex",
+        "ventrolateral prefrontal cortex",
+        "medial frontal cortex",
+        "primary motor cortex",
+        "primary somatosensory cortex",
+        "posteroinferior parietal cortex",
+        "primary auditory cortex",
+        "posterior superior temporal cortex",
+        "inferolateral temporal cortex",
+        "primary visual cortex",
+        "hippocampus",
+        "amygdaloid complex",
```

Comment:
http://purl.obolibrary.org/obo/UBERON_0001876 (amygdaloid complex is a synonym for amygdala)